spec/features/user_sign_in_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature "User sign in" do
+
+  let (:user) do
+    User.create(name: 'Ivan Sathianathan', username: 'ivan', email: 'ivan@test.com', password: '12345', password_confirmation: '12345')
```

Comment:
Metrics/LineLength: Line is too long. [135/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)